dapps/digital-twin-detail/src/components/container/Entry.vue

```diff
@@ -24,7 +24,7 @@
       :data-schema="entrySchema"
       :is-loading="$store.state.container.dispatcherStates.entries[name]"
       :name="name"
-      :value="getValue"
+      :value="getValue()"
```

Comment:
isn't it already updated? strange that it appears in the diff here ...

Reply:
Yes, but maybe because I added it manually just to get it to work without merging. Still strange that git doesn't notice this.